docs/en/connector-v2/source/FakeSource.md

```diff
@@ -16,54 +16,53 @@ just for some test cases such as type conversion or connector new feature testin
+| bigint.max          | bigint   | no       | 0x7fffffffffffffff      | The max value of bigint data that connector generated                                                                                                                                 |
+| bigint.template     | list     | no       | -                       | The template list of bigint type that connector generated, if user configured it, connector will randomly select an item from the template list                                       |
+| float.fake.mode     | string   | no       | range                   | The fake mode of generating float data, support `range` and `template`, default `range`，if use configured it to `template`, user should also configured `float.template` option       |
+| float.min           | float    | no       | 0                       | The min value of float data that connector generated                                                                                                                                  |
+| float.max           | float    | no       | 0x1.fffffeP+127         | The max value of float data that connector generated                                                                                                                                  |
+| float.template      | list     | no       | -                       | The template list of float type that connector generated, if user configured it, connector will randomly select an item from the template list                                        |
+| double.fake.mode    | string   | no       | range                   | The fake mode of generating float data, support `range` and `template`, default `range`，if use configured it to `template`, user should also configured `double.template` option      |
+| double.min          | double   | no       | 0                       | The min value of double data that connector generated                                                                                                                                 |
+| double.max          | double   | no       | 0x1.fffffffffffffP+1023 | The max value of double data that connector generated                                                                                                                                 |
+| double.template     | list     | no       | -                       | The template list of double type that connector generated, if user configured it, connector will randomly select an item from the template list                                       |
+| table-names         | list     | no       | -                       | The Supports configuration of multiple data sources for example: ["test.table1", "test.table2"]                                                                                       |
```

Comment:
Check the old one https://github.com/apache/seatunnel/pull/5255/files#diff-368d0e6bf6b91efe5c90ec4eaa163de7332b3e6e64797a0fc6b27dd3d0711326L266. You losted many important information in the new one description.